src/PhpSpreadsheet/Reader/Xlsx/ConditionalStyles.php

```diff
@@ -196,12 +196,19 @@ private function readStyleRules($cfRules, $extLst)
                 $objConditional->setStopIfTrue(true);
             }
 
-            if (count($cfRule->formula) > 1) {
-                foreach ($cfRule->formula as $formula) {
-                    $objConditional->addCondition((string) $formula);
+            if (count($cfRule->formula) >= 1) {
+                foreach ($cfRule->formula as $formulax) {
+                    $formula = (string) $formulax;
+                    if ($formula === 'TRUE') {
```

Comment:
I'm longing for the day when we've reached a minimum level of PHP that we can do:
```php
$objConditional->addCondition(
    match((string) $formulax) {
        'TRUE' => true,
        'FALSE' => false,
        default => $formula
    }
);
```
So much easier to read than `if`/`elseif`/`else`.

Does this need to make allowance for locale variants in TRUE/FALSE strings (VRAI/FAUX or VERO/FALSE)?